### ISSUE TITLE
Support for Missing SVG Font and Text Attributes

### ISSUE THREAD

**Issue by skibulk** [mikolalysenko/vectorize-text#5]
There are many font and text properties in the [SVG spec](https://www.w3.org/TR/SVG/styling.html). Several of them are missing from your list of supported options. Will you add support for the following?

Font properties:
‘font-size-adjust’
‘font-stretch’

Text properties:
‘direction’
‘letter-spacing’
‘text-decoration’
‘unicode-bidi’
‘word-spacing’
‘alignment-baseline’
‘baseline-shift’
‘dominant-baseline’
‘glyph-orientation-horizontal’
‘glyph-orientation-vertical’
‘kerning’
‘text-anchor’
‘writing-mode’

**Comment by mikolalysenko** [comment]
Yeah, it isn't comprehensive.  If you want to add a few more options I'll accept a PR.
